optashift-employee-rostering-gwtui/pom.xml

```diff
@@ -73,6 +73,16 @@
       <groupId>org.jboss.errai</groupId>
       <artifactId>errai-navigation</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.kie.soup</groupId>
+      <artifactId>kie-soup-commons</artifactId>
```

Comment:
@mbarkley This project doesn't depend on kie-wb nor appformer nor uberfire.
How come we need to import kie-soup?

Reply:
Errai requires the source code of `Precondition` (https://github.com/kiegroup/kie-soup/blob/master/kie-soup-commons/src/main/java/org/kie/soup/commons/validation/Preconditions.java). It doesn't get imported with the other dependencies currently.

Reply:
@mbarkley You're right, got confused since the log was full of Errai info messages before the error (and some Errai issues from using an incompatible gwt version which is now fixed)

Reply:
Does this mean we can remove the explicit dependency on kie-soup?

Reply:
I don't know as much about the new GWT maven plugin, but assuming it doesn't automatically add source jars, then @Christopher-Chianelli is correct that you'll need to explicitly add the source jar.